src/utils/dateFormatter.ts

```diff
@@ -0,0 +1,8 @@
+const formatPublishedDate = (dateString: string): string => {
```

Comment:
btw you can also make this a Date object (`new Date(...)`) the parse it using Date functions (getYear or toLocaleDateString or ...)

Reply:
tried, no success, the time published string goes like this 20240502T135923